packages/encryption-utils/src/index.ts

```diff
@@ -137,11 +152,16 @@ export default class EncryptionUtils {
     }
 
     static hash(data: any) {
-        if (typeof(data) != 'string') {
-            data = JSON.stringify(data)
+        
+        if (typeof(data) === 'string') {
+            if (!isHexString(data)) {
+                data = utils.toUtf8Bytes(data)
+            }
+        } else if (!isArrayLike(data)) {
```

Comment:
If the data is an array, it will no longer be JSON encoded. That is a change from the previous behaviour.

Is that intentional?

Reply:
Not only that. If data is hex string, it will not convert it to Utf8bytes.